### DISCUSSION TITLE
EventToCommandBehavior inside DataTemplates.xaml

### DISCUSSION THREAD

**Discussion by JozefGulaPayius** [CommunityToolkit/Maui#784] [Q&A]
Hello, 

I am trying to use EventToCommandBehavior in a DataTemplate.

If I use this binding in "TapGestureRecognizer", everything works correctly:

`Command="{Binding Path=EditCommand, Source={RelativeSource AncestorType={x:Type viewModels:MyViewModel}}}"`

The same binding in "EventToCommandBehavior" end with: 

`System.InvalidOperationException: 'Operation is not valid due to the current state of the object.`

![image](https://user-images.githubusercontent.com/20856854/203797837-dc900a3f-dbe2-46d8-8674-cd10c696a276.png)

Is this standard behavior or a bug? May someone please advise me how to solve this problem?

Thanks a lot

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
